pkg/testdata/models/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+service:
+  full_name: Amazon Test Service
+  short_name: ATS
+  link: https://example.com/
+  documentation: https://docs.example.com/
+versions:
+- api_version: v1alpha1
+  status: development
```

Comment:
It might be useful to have a helper script in the code-generator that will examine a new controller repository and construct a default metadata file given a service name...